packages/panels/resources/views/pages/dashboard.blade.php

```diff
@@ -1,7 +1,18 @@
 <x-filament-panels::page class="fi-dashboard-page">
+    @if (method_exists($this, 'filtersForm'))
+        <x-filament::section>
```

Comment:
Maybe we should add a heading to this section?

Reply:
I tried, but IMO it takes up too much vertical space